starter-code/contact-app/src/app/contact-list/contact-list.component.html

```diff
@@ -15,4 +15,14 @@
   <input type="text" [(ngModel)]="newContact.image">
 </div>
 
-<button (click)="addContact">Submit New Contact</button>
+<button (click)="addContact()">Submit New Contact</button>
```

Comment:
Better use the form structure that you have in the cheat sheet: https://github.com/ironhack/bcn-webdev-cheatsheet/tree/master/m3/angular-forms